task_processing/plugins/kubernetes/task_config.py

```diff
@@ -190,6 +190,11 @@ def __invariant__(conf):
         invariant=_valid_capabilities,
     )
 
+    @property
+    def prefix(self) -> str:
```

Comment:
are there any other places we could use this prefix outside of secrets? if not, maybe we should do a more specific name? e.g., `secret_X`?

Reply:
This is a good call, because we are really only ever going to use this for secrets, I've just skipped this altogether and passed the full task_config.name to our secret utils instead.